virt/virt-2-4-release-notes.adoc

```diff
@@ -195,5 +192,10 @@ nodes.
 running either `oc adm drain` or `kubectl drain`. Do not run these commands on
 the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
 drain if there are virtual machines running on top of them.
+<<<<<<< HEAD
```

Comment:
@lmandavi, please fix this merge conflict and squash.

And are you sure you're changing the link again?

Reply:
@kalexand-rh - this particular content shown here is not related to this PR, it was content belonging to another team member. I need to go back into the defect and see if this content is relevant and should be retained or deleted.